meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx_git.bb

```diff
@@ -1,21 +1,23 @@
-include recipes-kernel/linux/kmeta-linux-lmp-5.4.y.inc
+include recipes-kernel/linux/kmeta-linux-lmp-5.10.y.inc
 
-LINUX_VERSION ?= "5.4.149"
-KBRANCH = "5.4-2.3.x-imx"
-SRCREV_machine = "09a7977be879b5c2f055122df4992d2c8b7f2588"
+# Use Freescale kernel by default
+KERNEL_REPO ?= "git://github.com/Freescale/linux-fslc.git"
+KERNEL_REPO_PROTOCOL ?= "https"
+LINUX_VERSION ?= "5.10.90"
+KERNEL_BRANCH ?= "5.10-2.1.x-imx"
+
+SRCREV_machine = "e730e691a1edb8c38004d81b070529224e8df714"
 SRCREV_meta = "${KERNEL_META_COMMIT}"
 
-LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
-SRC_URI = "git://github.com/Freescale/linux-fslc.git;protocol=https;branch=${KBRANCH};name=machine; \
+SRC_URI = "${KERNEL_REPO};protocol=${KERNEL_REPO_PROTOCOL};branch=${KERNEL_BRANCH};name=machine; \
     ${KERNEL_META_REPO};protocol=${KERNEL_META_REPO_PROTOCOL};type=kmeta;name=meta;branch=${KERNEL_META_BRANCH};destsuffix=${KMETA} \
-    file://0001-FIO-fromtree-drivers-optee-allow-op-tee-to-access-de.patch \
-    file://0002-FIO-fromtree-hwrng-optee-handle-unlimited-data-rates.patch \
-    file://0003-FIO-fromtree-hwrng-optee-fix-wait-use-case.patch \
     file://0004-FIO-toup-hwrng-optee-support-generic-crypto.patch \
-    file://0001-FIO-fromlist-drivers-optee-i2c-add-bus-retry-configu.patch \
-    file://0001-FIO-fromtree-ath10k-add-QCA9377-sdio-hw_param-item.patch \
```

Comment:
I wonder if we should delete these patches from meta-lmp as well: https://github.com/foundriesio/meta-lmp/tree/12058e34ff9fe6cf80b0b81cae006a97ab9d2467/meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx

Reply:
Problem is if we got another recipe/layer/factory using them, but I'm +1 for removing if not used by meta-lmp (we can always add them back to specific factories if needed).

Reply:
Thanks, missed that :)